src/cascadia/TerminalControl/TermControlAutomationPeer.cpp

```diff
@@ -142,6 +143,28 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         });
     }
 
+    void TermControlAutomationPeer::NotifyNewOutput(const std::wstring_view newOutput)
+    {
+        auto dispatcher{ Dispatcher() };
+        if (!dispatcher)
+        {
+            return;
+        }
+
+        dispatcher

```

Comment:
I should add a comment here pointing out that we're using `.get()` to make this sequential.